src/status_im/ui/screens/screens.cljs

```diff
@@ -560,7 +562,7 @@
 
    ;[Chat] New Chat
    {:name      :new-chat
-    :on-focus  [::new-chat.events/new-chat-focus]
+    :on-focus  [::add-new-contact/new-chat-focus]
```

Comment:
we decided to do not use namespaced keywords

Reply:
removed namespaced keywords, thanks!